cluster/charts/rook-ceph/templates/resources.yaml

```diff
@@ -1908,6 +1908,13 @@ spec:
                   enum:
                   - image
                   - pool
+                snapshotSchedule:
```

Comment:
maybe you want to move this block after line 1115 with `nullable: true`. This is currently inside else block which is `v1beta1` for pre k8s1.16, So, I think we want this to place in `v1`. @travisn ??

Reply:
Oh, let me look closer. Thanks

Reply:
Why not both?

Reply:
We can keep for both but `v1beta1` is not that strict and I think we don't want to change much in `v1beta1`.

Reply:
But users can still deploy v1beta1 if they have an old kube cluster right?

Reply:
Yes right, actually I and Travis were discussing similar kind of situation yesterday on #6534 and there we decided to just keep the place holder for the `v1beta1`. So, I said with that respect.

Reply:
v1beta1 does not support the full schema settings like `nullable`. My thought is to not worry about updating the v1beta1 schema since it wasn't enforced anyway.